src/config.js

```diff
@@ -96,6 +96,27 @@ let commonConf = {
     default: 8080,
     env: 'PORT'
   },
+  db: {
```

Comment:
Should we take this as the default for Sensorthings as well? I mean, unless the user specifies a remote sensorthings config, this db configuration is used for deploying sensorthings api. What do you think?